types/gulp-modernizr/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for gulp-modernizr 1.0
+// Project: https://github.com/doctyper/gulp-modernizr
+// Definitions by: remisery <https://github.com/remisery>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function GulpModernizr(fileOrParams?: string | GulpModernizr.Params, parames?: GulpModernizr.Params): any;
+
+declare namespace GulpModernizr {
+    interface Params {
+        /**
+         * Avoid unnecessary builds (see Caching section below)
+         */
+        cache?: boolean;
+
+        /**
+         * Path to the build you're using for development.
+         */
+        devFile?: boolean;
```

Comment:
The type of devFile looks to be string | boolean by looking at: https://www.npmjs.com/package/customizr